### ISSUE TITLE
Tags not showing up in Loggly

### ISSUE THREAD

**Issue by git-ong** [MauriceButler/bunyan-loggly#3]
Tags are not making their way to Loggly.

In order to get my tags to appear in Loggly, I need to modify index.js line 60 from:

```
this.client.log(content);
```

To something like this:

```
this.client.log({ content: content }, this.logglyConfig.tags);
```

My test program:

```
var bunyan = require('bunyan'),
    Bunyan2Loggly = require('bunyan-loggly').Bunyan2Loggly,
    logger;

// create the logger 
var logger = bunyan.createLogger({
  name: 'logglylog',
  streams: [
    {
      type: 'raw',
      stream: new Bunyan2Loggly({
    token: 'your-account-token',
    subdomain: 'your-sub-domain',
    tags: [ 'global-tag' ]
      })
    }
  ]
});

logger.info("foobar");
```

**Comment by git-ong** [comment]
Fixed it by looping over the content array and sending each element separately to Loggly. Please see my pull request. Thanks!

**Comment by MauriceButler** [comment]
This appears to have been merged into master in commit 60743447905e61161b40a6ce368c69ce596ffe4f closing now

**Comment by git-ong** [comment]
+1. Thanks!
